bigchaindb/core.py

```diff
@@ -365,15 +285,7 @@ def validate_transaction(self, transaction):
                     raise exceptions.DoubleSpend('input `{}` was already spent'.format(
                         transaction['transaction']['input']))
 
-        # Check hash of the transaction
```

Comment:
does it make sense to group this logic? it does not seem to be used anywhere else